pair.h

```diff
@@ -5,11 +5,8 @@
 // and it comes with more 'modern C++' bells and whistles (most notably, piecewise construct)
 //#define RDESTL_USE_STD_PAIR 1
 
-#include "type_traits.h"
-
-#ifdef RDESTL_USE_STD_PAIR

```

Comment:
`std::move` depends on the `<utilities>` header also. Any reason not to implement move/forward/etc into rdestl directly, instead of relying on the std library for those?

Reply:
Main reason is it already exists and the implementation is pretty much the same as a custom one. I treat move/forward as part of the C++ std, not necessarily STL